spec/system/custom/admin/site_customization/pages_spec.rb

```diff
@@ -0,0 +1,29 @@
+require "rails_helper"
+
+describe "Admin custom pages", :admin do
+  context "Create" do
+    scenario "Allow create custom page with color fields" do
```

Comment:
How about a scenario for editing existing pages? That way we can test the color fields are shown when editing a page with an enabled custom header.